force-app/main/default/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -129,6 +124,43 @@
         $A.enqueueAction(action);
     },
 
+    /*******************************************************************************************************
+     * @description called at onInit to load up the Household Object/Account, and its Contacts
+     */
+    loadObjects: function(component) {
+        component.set("v.showSpinner", true);
+        var hhId = component.get('v.hhId');
+        
+        // handle new household object
+        if (hhId === null)
+            return;
+        
+        component.set('v.hhTypePrefix', String(hhId).substr(0, 3));
+        var namespacePrefix = component.get('v.namespacePrefix');
+
+        this.loadHousehold(component, hhId, namespacePrefix);
+        this.loadContacts(component, hhId, namespacePrefix);
+        this.loadSalutations(component);
+        this.loadHouseholdDeletePermissions(component);
+        this.loadAddresses(component, hhId, namespacePrefix);
+        this.loadFieldLabels(component, namespacePrefix);
+    },
+
+    loadFieldLabels: function(component) {
```

Comment:
Thank you for not adding additional Custom Labels, I'm very curious why the other fields are setup that way. Maybe because Address is a compound field?